.eslintrc.js

```diff
@@ -23,13 +32,10 @@ module.exports = {
       }
     ],
     '@typescript-eslint/await-thenable': 'off',
-    '@typescript-eslint/ban-ts-ignore': 'error',
-    '@typescript-eslint/ban-types': 'error',
+    '@typescript-eslint/ban-types': 'off',
```

Comment:
Wy turning this rule off? Seems like a sane list https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md#default-options

Reply:
`
Small question, should I enable '@typescript-eslint/ban-types': 'off' to error as it was before? Tho I'll have to change some of the crucial code, since newer ban-types doesn't allow object type, and requires more specific Record type (which, I think is actually better than pure object)
`
I was asking bout it coz it was changed a bit and it doesn't like `object`

Reply:
Ah, I was wondering why I see the `Record` type now in the code. Let me see...